wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/E2EIdentityCertificateStatus.swift

```diff
@@ -34,8 +31,6 @@ extension E2EIdentityCertificateStatus {
             return L10n.Localizable.Device.Details.Section.E2ei.Status.expired
         case .valid:
             return L10n.Localizable.Device.Details.Section.E2ei.Status.valid
-        case .none:
```

Comment:
**suggestion:** I would rename `func titleForStatus()` and `func imageForStatus()` to `var title` and ` var image`. And make it as a property. They are both for the `E2EIdentityCertificateStatus`, so why don't repeat status?

Reply:
done